src/hooks/useCollectionContext.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactChild, useReducer } from 'react'
+import { collectionReducer } from 'utils'
+
+const MyCollectionContext = React.createContext<{
```

Comment:
```suggestion
const CollectionContext = React.createContext<{
```

I would match the provider/file name